### PR TITLE
tutor:0.3.0

### PR DESCRIPTION
**Pull request by rangerjo**
I am submitting
- [x] a new package
- [ ] an update for a package

Utilities to write exams and exercises with integrated solutions. Set the variable `sol=true` to display the solutions of a document. This package can be useful to teachers and alike.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

### CODE REVIEW

File: packages/preview/tutor/0.3.0/README.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+* Checkboxes that are either blank or show the solution state using eg. `#checkbox(cfg, true)`.
+* Display blank lines allowing students to write their answer using eg. `#lines(cfg, 3)`.
+* A proposition for a project structure allowing self-contained exercises and a mechanism to show or hide the solutions of an exercise.
+
+## Usage
+
+### Minimal Example
+
+```typst
+#import "@local/tutor:0.2.0": points, totalpoints, lines, checkbox
```
Comment (laurmaedje) [inline packages/preview/tutor/0.3.0/README.md:17]: This (and other examples) should show the `@preview` import at the 0.3.0 version.

File: packages/preview/tutor/0.3.0/README.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+#let cfg = default_config()
```
Comment (laurmaedje) [inline packages/preview/tutor/0.3.0/README.md:19]: The idiomatic case would be `default-config` in case you want to change that. (It's up to you.)

Comment (rangerjo) [reply packages/preview/tutor/0.3.0/README.md]: Still have to get used to kebab-case, but thanks for the hint. Please find the updates below.

**Comment by laurmaedje** [comment]
Thank you!

**Pull request merged**